crates/api/src/op_store.rs

```diff
@@ -68,7 +68,7 @@ pub trait OpStore: 'static + Send + Sync + std::fmt::Debug {
     /// if it is able to process them.
     fn process_incoming_ops(
         &self,
-        op_list: Vec<MetaOp>,
+        op_list: Vec<Op>,
```

Comment:
I probably wouldn't choose to use the wire types in the API. Could we use `bytes::Bytes` here?

Reply:
I follow your inclination here =)
No, makes perfect sense to not use the wire representation.